utils/shonenjump/patches/0001-store.go-do-not-call-Age.patch

```diff
@@ -0,0 +1,24 @@
+From a6b29a74b492f8792c74304081e454ef9f314324 Mon Sep 17 00:00:00 2001
+From: graysky <therealgraysky@proton.me>
+Date: Tue, 1 Nov 2022 07:44:06 -0400
+Subject: [PATCH] store.go: do not call Age()
+
+Disable aging function to more closely approximate autojump behavior.
+
+See: https://github.com/suzaku/shonenjump/issues/10
+
+---
+ jump/store.go | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+--- a/jump/store.go
++++ b/jump/store.go
```

Comment:
I don't think this patch should be applied unless it is accepted into upstream. This currently represents your preference; other users of shonenjump may not share this preference and there is no way for them to reverse this change.